rules_ruby_gapic/gapic_src_repo.bzl

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+A function that loads gapic-generator-ruby itself as a dependency to build the libraries with
+and then strips it from the softlinks so that bazel accepts it
+"""
+load("//rules_ruby_gapic/ruby:private/utils.bzl", _execute_and_check_result = "execute_and_check_result")
+
+##
+# An implementation of gapic_generator_src rule
+#
+def _gapic_generator_src_impl(ctx):
+  ctx.download_and_extract(
+    url = "https://github.com/viacheslav-rostovtsev/gapic-generator-ruby/archive/45d1159fe395881ad6e851e4c91bd65a332c9fef.zip",
```

Comment:
Can we use a release in the main repo instead? Go ahead and make one if we need to.